_tour/abstract-types.md

```diff
@@ -1,4 +1,4 @@
----
+
```

Comment:
This seems accidental. It would break the Abstract Types page, I think.

Reply:
That's probably because I copied the headers from English versions.. :smile:
Three dashes aren't missing. It's just that git-diff rendered them after deleted section.  
I'm using Bash on Windows 10 and it renders ok.
If there are errors when merged I'll fix it.